pkg/kdump/kdump-view.jsx

```diff
@@ -41,14 +41,74 @@ import { Tooltip, TooltipPosition } from "@patternfly/react-core/dist/esm/compon
 import { OutlinedQuestionCircleIcon } from "@patternfly/react-icons";
 
 import { useDialogs, DialogsContext } from "dialogs.jsx";
+import { read_os_release } from "os-release.js";
 import { fmt_to_fragments } from 'utils.jsx';
 import { show_modal_dialog } from "cockpit-components-dialog.jsx";
 import { FormHelper } from "cockpit-components-form-helper";
 import { ModalError } from 'cockpit-components-inline-notification.jsx';
 import { PrivilegedButton } from "cockpit-components-privileged.jsx";
+import { ModificationsExportDialog } from "cockpit-components-modifications.jsx";
 
 const _ = cockpit.gettext;
 
+const exportAnsibleTask = (settings, os_release) => {
+    const target = Object.keys(settings.targets)[0];
+    const targetSettings = settings.targets[target];
+    const kdump_core_collector = settings.core_collector;
+
+    let role_name = "linux-system-roles";
+    let galaxy = `
+# ${role_name} is also available via ansible-galaxy https://galaxy.ansible.com/ui/standalone/roles/linux-system-roles/kdump/`;
+
+    if (os_release.NAME === "RHEL" || (os_release.ID_LIKE && os_release.ID_LIKE.includes('rhel'))) {
+        role_name = "rhel-system-roles";
+        galaxy = "";
+    }
+
+    let ansible = `
+---
+${galaxy}
+- name: install ${role_name}
+  package:
+    name: ${role_name}
+    state: present
+  delegate_to: 127.0.0.1
+  become: true
+- name: run kdump system role
+  include_role:
+    name: ${role_name}.kdump
+  vars:`;
+
+    if (target === "local") {
+        ansible += `
+    kdump_path: ${targetSettings.path || "/var/crash"}
+    kdump_core_collector: ${kdump_core_collector}
+`;
+    } else if (target === "ssh") {
+        const ssh_user = targetSettings.server.split('@')[0];
+        ansible += `
+    kdump_target:
+      type: ssh
+      location: ${targetSettings.server}
+    kdump_path: ${targetSettings.path || "/var/crash"}
+    kdump_sshkey: ${targetSettings.sshkey}
+    kdump_ssh_user: ${ssh_user}
+    kdump_ssh_server: ${targetSettings.server}
+    kdump_core_collector: ${kdump_core_collector}
+`;
+    } else if (target === "nfs") {
+        ansible += `
+    kdump_target:
+      type: nfs
+      location: ${targetSettings.server}:${targetSettings.export}
+    kdump_path: ${targetSettings.path || "/var/crash"}
+    kdump_core_collector: ${kdump_core_collector}
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19430-20231120-143451-169a6c2c-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)